packages/babel-parser/test/fixtures/es2019/json-strings/template-line-separator/input.js

```diff
@@ -0,0 +1,2 @@
+(`before after`);
```

Comment:
The behaviour of U+2028 in string template is _not_ changed. Adding test here for coverage.